recipes/log4cxx/recipe/meta.yaml

```diff
@@ -0,0 +1,69 @@
+#--------------------------------------------------------------------------------
+# MetTools - A Collection of Software for Meteorology and Remote Sensing
+# Copyright (C) 2018  EUMETSAT
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#--------------------------------------------------------------------------------
+
+package:
+  name: log4cxx
+  version: 0.10.0
+
+source:
+  url: http://archive.apache.org/dist/logging/log4cxx/0.10.0/apache-log4cxx-0.10.0.tar.gz
+  fn:  apache-log4cxx-0.10.0.tar.gz
+  sha256: 0de0396220a9566a580166e66b39674cb40efd2176f52ad2c65486c99c920c8c
+  patches:
+    - 001-fix-compilation.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - expat 2.2.*
+    - apr 1.6.*
+    - libiconv 1.14
```

Comment:
Isn't this working with 1.15 as in the global pinning?

Reply:
worked locally, let's see.